src/lenskit/data/builder.py

```diff
@@ -165,8 +169,9 @@ def add_relationship_class(
         self,
         name: str,
         entities: RelationshipEntities,
-        allow_repeats: bool = True,
+        repeats: bool = True,
         interaction: bool = False,
+        remove_duplicates: bool | Literal["exact"] = False,
```

Comment:
We went back and forth on a couple designs here, but where we ultimately landed is that whether repeats are _allowed_ is a property of the relationship class that gets stored in the schema, but whether they are _removed_ is a property of adding relationships. Therefore, `remove_duplicates` should be an option to `add_relationships` and `add_interactions`, but _not_ `add_relationship_class` (or `RelationshipSchema`).